sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2_and_later/async_samples/sample_send_request_async.py

```diff
@@ -43,7 +43,82 @@ async def sample_send_request():
     async with client:
         response = await client.send_request(request)
     response.raise_for_status()
+    response_body = response.json()
+    print(
+        f"Our resource has {response_body['customDocumentModels']['count']} custom models, "
+        f"and we can have at most {response_body['customDocumentModels']['limit']} custom models."
+        f"The quota limit for custom neural document models is {response_body['customNeuralDocumentModelBuilds']['quota']} and the resource has"
+        f"used {response_body['customNeuralDocumentModelBuilds']['used']}. The resource quota will reset on {response_body['customNeuralDocumentModelBuilds']['quotaResetDateTime']}"
+    )
+    
+    # pass with full url and override the api version
+    request = HttpRequest(method="GET", url=f"{client._endpoint}/formrecognizer/info?api-version=2022-08-31")
+    async with client:
+        response = client.send_request(request)
+    response.raise_for_status()
+    response_body = response.json()
+    print(
+        f"Our resource has {response_body['customDocumentModels']['count']} custom models, "
+        f"and we can have at most {response_body['customDocumentModels']['limit']} custom models."
+        f"The quota limit for custom neural document models is {response_body['customNeuralDocumentModelBuilds']['quota']} and the resource has"
+        f"used {response_body['customNeuralDocumentModelBuilds']['used']}. The resource quota will reset on {response_body['customNeuralDocumentModelBuilds']['quotaResetDateTime']}"
+    )
+    
+    # override the api version to v2.1
+    request = HttpRequest(method="GET", url="v2.1/custom/models?op=summary&api-version=v2.1")
```

Comment:
Is there a reason the API version is being specified with both a path and query parameter? Shouldn't we just use the former? Although, if we don't specify a query API version, I think the request would ultimately get send with `v2.1` in the path and the client's configured API version in the query... which we don't want to do.

Reply:
It's not necessary to specify API version in query, will update.